README.md

```diff
@@ -46,3 +46,32 @@ Where:
     --no-auto-ack : option to disable automatic acknowledgement of sync and notification packets
 
 For a list of supported commands, type "h" at the prompt
+
+### C
+
+#### orp
+
+A simple binary command-line utilty which can be used to mimic an ORP client, for testing and demonstration.
+
+Depends on:
+- clients/c/src/*
+- clients/c/inc/*
+
+Tested on:
+- Ubuntu 20.04
+
+Build:
+1. Install gcc
```

Comment:
No need to indicate a gcc minimum version ?

Reply:
Thanks.  I tested with gcc version 7 and version 9.  Have updated the doc to show this.